CRM/Contact/BAO/Contact.php

```diff
@@ -752,7 +752,7 @@ protected static function contactTrash($contact): bool {
     $contact->save();
     CRM_Core_BAO_Log::register($contact->id, 'civicrm_contact', $contact->id);
 
-    CRM_Utils_Hook::post('update', $contact->contact_type, $contact->id, $contact);
+    CRM_Utils_Hook::post('edit', $contact->contact_type, $contact->id, $contact);
```

Comment:
@eileenmcnaughton git blames you but you didn't write this you just moved it. But FYI it was wrong - there is no such action as "update" for pre/post hooks.

Reply:
I blame git right back!